lib/index.js

```diff
@@ -135,21 +145,34 @@ class Suite {
 		}
 		this.#plugins = options?.plugins || [new V8NeverOptimizePlugin()];
 
-		// Benchmark Mode setup
-		this.#benchmarkMode = options.benchmarkMode || "ops"; // Default to 'ops'
+		this.#benchmarkMode = options.benchmarkMode || "ops";
 		validateBenchmarkMode(this.#benchmarkMode, "options.benchmarkMode");
 
 		this.#reporterOptions = options.reporterOptions || {
 			printHeader: true,
 		};
 
-		// Suite-level minSamples option
+		this.#ttest = options.ttest || false;
```

Comment:
`options.ttest ?? false`

also if you’re validating all you other inputs should this one be as well?